src/Compilers/Core/Portable/Collections/ConcurrentCache.cs

```diff
@@ -16,6 +16,8 @@ namespace Microsoft.CodeAnalysis
     internal class ConcurrentCache<TKey, TValue> :
         CachingBase<ConcurrentCache<TKey, TValue>.Entry> where TKey : IEquatable<TKey>
     {
+        private readonly IEqualityComparer<TKey> _keyComparer;
```

Comment:
why was this needed?  it doesn't seem strictly necessary as part of this PR.

Reply:
Need to pass a key comparer